SourceGenerators/SourceGenerators - Backup.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please make Visual Studio stop creating "backup" files.

Reply:
Open Visual Studio and go to "Tools" -> "Options".

In the "Options" window, expand the "Projects and Solutions" node and select the "Build and Run" option.
Under the "MSBuild project build output verbosity" section, change the setting from "Minimal" to "Normal" 
Click on "OK" to save the changes.

Reply:
Added in f1eb0286